src/util/misc/promisfied-fs.ts

```diff
@@ -173,9 +173,24 @@ export async function walk(dir: string): Promise<string[]> {
   }
 }
 
+export function createLongPath(target: string) {
```

Comment:
Why is this completely synchronous function in a file called "promisified-fs"?

Going forward I want to get rid of this file, current versions of node support a native Promisify method that should easily replace all these wrappers. But that's for later.

Reply:
@erychagov can we make this function `async`? Or has it been replaced by `fs-helper.ts`? Have we had a bad merge (may be my fault?) I'm sure you sorted this previously.